mozfldp/server.py

```diff
@@ -24,74 +19,120 @@ class ServerFacade:
     This class acts as a facade around the functions in
     `simulation_util` to provide a consistent interface to load data
     into a classifier.
+
+    Args: 
+        coef: initial coefficients to initialize the server with
+        intercept: initial intercepts to initialize the server with
+        num_client: number of clients that are available
+        client_fraction: (C in algorithm) - fraction of clients that we want to use in a round
     """
 
-    def __init__(self):
-        self._classifier = SGDClassifier(loss="hinge", penalty="l2")
+    def __init__(self, coef, intercept, num_client, client_fraction):
```

Comment:
I don't think we need `num_client` and `client_fraction` to be maintained as member variables in the Server class. AFAICT they are not used anywhere else in here. Selecting the fraction of clients to update is handled externally by a runner like `FLSimulationRunner`

Reply:
Done